crates/ruff_linter/src/rules/flake8_implicit_str_concat/rules/implicit.rs

```diff
@@ -34,6 +34,12 @@ use crate::Locator;
 /// ```python
 /// z = "The quick brown fox."
 /// ```
+///
+/// ## Formatter compatibility
+/// We recommend against using this rule alongside the [formatter]. The
+/// formatter enforces consistent use of implicit string concatenation, making the rule redundant.
```

Comment:
The rule is actually needed when using the formatter because the formatter can introduce new single-line implicitly concatenated strings. There's more detail about the incompatibility in https://github.com/astral-sh/ruff/issues/8272 

We'll make this rule compatible with the new 2025 style, so maybe it's not worth documenting it for now.

Reply:
I added this compatibility note due to the rule `ISC001` being on the list of incompatible linting rules: https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules.
Perhaps a better message would be:
```rust
/// # Formatter compatibility
/// and a section body. There should be no blank lines between a section header
/// Use of this rule alongside the [formatter] must be handled with care.
/// Currently, the [formatter] can introduce new single-line implicitly
/// concatenated strings, therefore we suggest rerunning the linter and
/// [formatter] in the following order:
/// 1. Run the linter with this rule (`ISC001`) disabled
/// 2. Run the [formatter]
/// 3. Rerun the linter with this rule (`ISC001`) enabled
/// This is one of very few cases where the [formatter] can produce code that
/// contains lint violations. It is a known issue that should be resolved by the
/// new 2025 style guide.
```

Reply:
This sounds great

Reply:
Do you have a link to the documentation for what will change with the 2025 style?